tests/common/dualtor/data_plane_utils.py

```diff
@@ -144,13 +144,21 @@ def run_test(duthosts, activehost, ptfhost, ptfadapter, action,
     send_and_sniff.set_error_handler(lambda *args, **kargs: io_ready.set())
 
     send_and_sniff.start()
+    io_ready.wait()
     if action:
         # do not perform the provided action until
         # IO threads (sender and sniffer) are ready
-        io_ready.wait()
         logger.info("Sender and sniffer threads started, ready to execute the "\
             "callback action")
         action()
+    # do not time-wait the test, if early start is not requested (when stop_after=0)
+    if stop_after:
```

Comment:
Since the default `stop_after` value is `None` now, I think this needs to be changed to `if stop_after is not None:` if we want to allow a 0 second delay.

Reply:
Right, added the change now.